src/librustc_llvm/lib.rs

```diff
@@ -2149,6 +2151,12 @@ pub fn SetLinkage(global: ValueRef, link: Linkage) {
     }
 }
 
+pub fn SetUniqueComdat(llmod: ModuleRef, val: ValueRef) {
+    unsafe {
+        LLVMRustSetComdat(llmod, val, LLVMGetValueName(val));
```

Comment:
this function needs a comment; at minimum, it should link to that blog post, but also: why does it make sense to use `LLVMGetValueName` here? 

I would expect the string to be the symbol name or something similar -- does that happen to be what `LLVMGetValueName` is returning here?